test/bn128.js

```diff
@@ -76,15 +75,15 @@ describe("Basic tests for g1 in bn128", () => {
 
         bn128.g1_fromMontgomery(p1, p1);
 
-        assert(bn128.g1_isZero(p1));
+        assert.equal(1, bn128.g1_isZero(p1));
```

Comment:
g1_isZero returns 1 or 0, not boolean?

Reply:
That is correct.  I agree that `is*` functions should return a boolean, but I don't think it is worth changing that behavior at the moment.